### DISCUSSION TITLE
Loki: Query builder should recognize if things are escaped and use  " instead of `

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#59541] [Data Sources]
Query builder should recognize if things are escaped in the regex, and use "" instead of `. For example in this query:  
```
{server_type="server1"} |= `conetxtId=1` | regexp `conetxtId=(?P<root_id>\\S*),` | regexp ` conetxtId=(?P<trace_id>\\S*)`
```

it should recognize that things are escaped and use "

```
{server_type="server1"} |= `conetxtId=1` | regexp "conetxtId=(?P<root_id>\\S*)," | regexp "conetxtId=(?P<trace_id>\\S*)"
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
